### DISCUSSION TITLE
Craft 5 migration should report back what entry types were renamed

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#15924] [Ideas] [answered]
After running a Craft 5 migration, there's a good chance the user will end up with entry type name collisions from matrix blocks resulting in entry types like: video, video2, video3. Rather than silently appending a number, the migration should report back what changes were made to give the developer a chance to make template changes if needed.

Less likely to be an issue, it might not be a bad idea to do the same thing for fields.

**Comment by brandonkelly** [comment, marked as answer]
Did you one better! As of Craft 5.6, entry types’ names and handles can now be overridden on a per-usage basis (sections and Matrix fields). Entry type names are no longer changed during the upgrade (since they stopped needing to be unique in 5.3), and their handles are automatically overridden to match their Craft 4 counterparts. (#16453)

So there won’t really be anything worth reporting anymore; your templates should continue to work as expected (with the exception of `type` entry query params, which will still need to reference the updated handles).

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out now 🚀
